### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by corymj**
Simply added updated reply language for petabencana.
(Full refactor push to come)

### CODE REVIEW

File: handler.js [hunk 1]
```diff
@@ -27,19 +27,22 @@ const instance_regions = {
...
+  'id': 'Selamat datang! ketik /banjir untuk meminta link kartu',
+  'in': 'Hai!!  Saya BencanaBot. Tekan menu di bawah atau ketik ’Laporkan banjir'
 }
 
 // Replies to user
 const replies = {
   'en': 'Hi! Report using this link, thanks.',
-  'id': 'Hi! Laporkan menggunakan link ini. Terima kasih.'
+  'id': 'Hi! Laporkan menggunakan link ini. Terima kasih.',
+  'in': 'Hi! Laporkan menggunakan link ini. Terima kasih.'
```
Comment (asbarve) [inline handler.js]: lets only keep en and id for the moment - will create India specific languages later (ta:tamil)

File: sample.env [hunk 2]
```diff
@@ -1,8 +1,8 @@
 VALIDATIONTOKEN=token used to validate messenger webhooks
-APPSECRET= appsecret key got on creating facebook app
-PAGEACCESSTOKEN= access token for the page to which the app has subscribed
-FACEBOOKSNS= sns topic name for facebok
-DEFAULT_LANG=en
+APPSECRET=app secret key got on creating facebook app
+PAGEACCESSTOKEN=access token for the page to which the app has subscribed
+FACEBOOKSNS=sns topic name for facebok
+DEFAULT_LANG=in
```
Comment (asbarve) [inline sample.env]: for petabencana bot it should be id/ for all others it should be en

File: handler.js (hunk 1 above)
Comment (corymj) [reply handler.js]: done

File: sample.env (hunk 2 above)
Comment (corymj) [reply sample.env]: done (forgot to respond here in the comments)

**Pull request merged**